+NeuroAnalysis/+IO/MetaTable.m

```diff
@@ -0,0 +1,117 @@
+classdef MetaTable < handle
```

Comment:
honestly, i am not a fan of matlab object-oriented things. i thought we only need an table, and some functions working on the table. but i see you added a map alongside the tests. can you elaborate why this map is needed?